### PR TITLE
Default Platform for serve command

### LINKED ISSUE #408: 1.5.0 serve not working
**Description**
I'm having issues with serve command. It's just trying to launch last built executable instead.

```
wails serve
Wails v1.5.0 - Serving Application

> Ensuring Dependencies are up to date...
> Packing + Compiling project...
Awesome! Project 'My Project' built!
Serving Application: S:\Gitea\wailstest\my-project\build\my-project
Error: exec: "S:\\Gitea\\wailstest\\my-project\\build\\my-project": file does not exist
```

**To Reproduce**
Steps to reproduce the behaviour:
1. wails init (and select anything, it doesn't matter)
2. wails serve

**Expected behaviour**
Wait for the frontend to connects.
1.0.3-pre2 was working fine.

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v1.5.0 |
| Go Version    | go1.14.2 |
| Platform      | windows |
| Arch          | amd64 |
| GO111MODULE   | (Not Set) |
| GCC           | 8.1.0 |
| Npm           | 6.13.7 |
| Node          | v13.8.0|

### PR DESCRIPTION
**Pull request by dedo1911**
Fixes #408

### CODE REVIEW

**Comment by leaanthony** [comment]
This is great! could you please update the PR so it's against `develop`?

**Comment by dedo1911** [comment]
My bad. Should be correct now

File: cmd/cmd-mewn.go [hunk 1]
```diff
@@ -5,6 +5,6 @@ package cmd
...
...1c4d02bf34e5bbe2c0801f5cf1d8ce87312682eee7660dec1a82fff20ac0438f301bf9eb0f09b3c9f0ce37ea2f1dd776801ebc5639fafb2f11c1756da1f851d97d4689c520cdb8ed742c0b348239dcbbb683757fc64a865285201f9f38ae3f0736832b3d1cfca663c662fb6e6b8e4d06c1d69bffe74d24b4f38fda9ad3af00f62ef82ec2c2a03e1dd2440112e527f4240f10d1a9ed025a3084a926daf770b808d2638c7984abc661d44b6b98e37f67fe73a016743f2699b9d8dda28b10f95ffef7020c50c24bf86fb38f552a4a28cd4ca242327b16140641af677be150e8ae282716889a27c654d0cec383b9edfa71e102d6b6c8377e88ec6b107e1db18f3f3f7ab0e3c4dfdd49bae54fefe93e4d066d662de09238473f44c4a7c929d07e2804498417ff49f4ecc53a78dd67247b3627f3ada858d8714c127c7993f88957ed299f700de1df7c3f0a169f31bcf3d50e2f7218183de4377301bfd60f37c3c93c94871dee97fed6fd9efb6dcb07441b848a61f9406e1690f32788e0085da20f498614840a5f422a35f405a5f678673f5cf84bba075e089adc43fd2892824dd3cfda07a1d4a3507b633731c73f95ec1d67657f495102ef988745a1f603fa9dada75b2281251313fa4a6d8f1ede91881e0db9a0025983568bbe3e3944ea695f14cc2db3addf620bf3469ae0743a1dffb4e4f9d8e9745ac418e61caca07b0d01e3d182f8167f370725a3debf1f85f60365e08b620223fcc06884d7e09fee313f9e34f6b17e7980ab47d50c87b2d8c930762a5f47703d09ce7712fbe096adad4c48c3c6b15d2efa6981568fff6f000000ffffc3c33335cf1e0000")
```
Comment (leaanthony) [inline cmd/cmd-mewn.go:8]: Why is this file marked as changed?

Comment (dedo1911) [reply cmd/cmd-mewn.go:8]: I've honestly no clue. I think has something to do with changing branch, but the content looks the same. I can't see why it's being highlighted

Comment (leaanthony) [reply cmd/cmd-mewn.go:8]: Ah, that sucks, but not a deal breaker :-)

**Pull request merged**